### DISCUSSION TITLE
Ερώτηση για Θέμα 1 Ερωτ 2

### DISCUSSION THREAD

**Discussion by afoscolou** [sdy60/syllabus#84] [Q&A] [answered]
Στο Ερωτ 1 αναφέρεστε στη βασική ομάδα χρηστών και ζητάτε περιγραφή μιας persona. Θα περιγράψουμε 1 persona;

**Comment by epidrome** [comment, marked as answer]
ναι μια περσόνα, ο οδηγός-επισκέπτης του αυτοκινούμενου
